dll/win32/getuname/lang/ro-RO.rc

```diff
@@ -3662,262 +3662,262 @@ BEGIN
+    4378 "Litera choseong rieul-hieuh"
+    4379 "Litera choseong kapyeounrieul"
+    4380 "Litera choseong mieum-pieup"
+    4381 "Litera choseong kapyeounmieum"
+    4382 "Litera choseong pieup-kiyeok"
+    4383 "Litera choseong pieup-nieun"
+    4384 "Litera choseong pieup-tikeut"
+    4385 "Litera choseong pieup-sios"
+    4386 "Litera choseong pieup-sios-kiyeok"
+    4387 "Litera choseong pieup-sios-tikeut"
+    4388 "Litera choseong pieup-sios-pieup"
+    4389 "Litera choseong pieup-ssangsios"
+    4390 "Litera choseong pieup-sios-cieuc"
+    4391 "Litera choseong pieup-cieuc"
+    4392 "Litera choseong pieup-chieuch"
+    4393 "Litera choseong pieup-thieuth"
+    4394 "Litera choseong pieup-phieuph"
+    4395 "Litera choseong kapyeounpieup"
+    4396 "Litera choseong kapyeounssangpieup"
+    4397 "Litera choseong sios-kiyeok"
+    4398 "Litera choseong sios-nieun"
+    4399 "Litera choseong sios-tikeut"
+    4400 "Litera choseong sios-rieul"
+    4401 "Litera choseong sios-mieum"
+    4402 "Litera choseong sios-pieup"
+    4403 "Litera choseong sios-pieup-kiyeok"
+    4404 "Litera choseong sios-ssangsios"
+    4405 "Litera choseong sios-teung"
+    4406 "Litera choseong sios-cieuc"
+    4407 "Litera choseong sios-chieuch"
+    4408 "Litera choseong sios-khieukh"
+    4409 "Litera choseong sios-thieuth"
+    4410 "Litera choseong sios-phieuph"
+    4411 "Litera choseong sios-hieuh"
+    4412 "Litera choseong chitueumsios"
+    4413 "Litera choseong chitueumssangsios"
+    4414 "Litera choseong ceongchieumsios"
+    4415 "Litera choseong ceongchieumssangsios"
+    4416 "Litera choseong pansios"
+    4417 "Litera choseong ieung-kiyeok"
+    4418 "Litera choseong ieung-tikeut"
+    4419 "Litera choseong ieung-mieum"
+    4420 "Litera choseong ieung-pieup"
+    4421 "Litera choseong ieung-sios"
+    4422 "Litera choseong ieung-pansios"
+    4423 "Litera choseong ssangieung"
+    4424 "Litera choseong ieung-cieuc"
+    4425 "Litera choseong ieung-chieuch"
+    4426 "Litera choseong ieung-thieuth"
+    4427 "Litera choseong ieung-phieuph"
+    4428 "Litera choseong yesieung"
+    4429 "Litera choseong cieuc-ieung"
+    4430 "Litera choseong chitueumcieuc"
+    4431 "Litera choseong chitueumssangcieuc"
+    4432 "Litera choseong ceongchieumcieuc"
+    4433 "Litera choseong ceongchieumssangcieuc"
+    4434 "Litera choseong chieuch-khieukh"
+    4435 "Litera choseong chieuch-hieuh"
+    4436 "Litera choseong chitueumchieuch"
+    4437 "Litera choseong ceongchieumchieuch"
+    4438 "Litera choseong phieuph-pieup"
+    4439 "Litera choseong kapyeounphieuph"
+    4440 "Litera choseong ssanghieuh"
+    4441 "Litera choseong yeorinhieuh"
+    4442 "Litera choseong kiyeok-tikeut"
+    4443 "Litera choseong nieun-sios"
+    4444 "Litera choseong nieun-cieuc"
+    4445 "Litera choseong nieun-hieuh"
+    4446 "Litera choseong tikeut-rieul"
+    4447 "Litera choseong caracter de umplere"
+    4448 "Litera jungseong caracter de umplere"
```

Comment:
The same here. I meant "Litera hangul jungseong ...", as not mentioning "hangul" at all may be wrong.